docs/src/tutorials/standalone/Soil/freezing_front.jl

```diff
@@ -69,7 +69,7 @@
 
 # `` ϑ_l(t = 0, z) = 0.33 ``.
 
```

Comment:
First link in the file (to the other tutorial) does not work https://clima.github.io/ClimaLand.jl/previews/PR1413/generated/standalone/Soil/freezing_front/@ref%20Soil/soil_energy_hydrology.md